resources/src/data.rs

```diff
@@ -0,0 +1,18 @@
+use base64string::Base64String;
+
+/// In the Stellar network, key/value pairs can be attached to accounts.
+/// These key/value pairs can be useful for associating data with an account
+/// for various reasons. Data represents the value of a single key/value pair.
+///
+/// <https://www.stellar.org/developers/horizon/reference/resources/data.html>
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Data {
```

Comment:
I know that stellar calls this "data" but is it more like a datum?

Reply:
Very true